### ISSUE TITLE
Incorporate open civic data municipal scrapers?

### ISSUE THREAD

**Issue by derekeder** [opengovernment/askthem-legacy#17]
These guys seem to be pushing on some of the same wheels we are. Definitely worth checking out: https://github.com/opencivicdata/municipal-scrapers-us

**Comment by davidmooreppf** [comment]
Yep of course that's the Sunlight Open States peeps, w/ the support from Google.org to bring down to municipal - that would work for me, as another source for city councilmembers. We can check in w/ James & Paul directly of course. Good find, I hadn't seen their dozen or so cities yet.

**Comment by walter** [comment]
I took a look at the scrapers. They rely on the python version of [pupa](https://github.com/opencivicdata/pupa). After playing with the python version, my preference is for OpenNorth's fork of [pupa in ruby](https://github.com/opennorth/pupa-ruby).

The scrapers are city by city and each city's coverage varies (some have events/meetings and councilors, others have only one aspect). It's probably not going to be a "bulk" solution to our municipal data needs as it stands.

However they could be a building block for a toolkit that we distribute to municipal opendata developers to help us extend our dataset.

In short, worth keeping an eye and possibly useful in the long run.

**Comment by walter** [comment]
Currently planning to replace scraped_local_gov use with opencivicdata municipal scrapers.

Decided to stick with the python version of pupa since all the US municipal scrapers are done in python.

**Comment by walter** [comment]
Will be used to do the work for opengovernment/product/issues/19.
